### PR TITLE
Feature/harvey 1236 configure outputs v2

### PR DESCRIPTION
**Pull request by cesar-lopez-mar**
I made an alternative version that avoids double code

### CODE REVIEW

**Comment by cesar-lopez-mar** [comment]
Ready for review

File: src/xdainterface.cpp [hunk 1]
```diff
@@ -235,6 +235,40 @@ bool XdaInterface::connectDevice()
...
+        configArray.push_back(XsOutputConfiguration(XDI_RateOfTurn, sampling_frequency));
+      if (enable_orientation)
+        configArray.push_back(XsOutputConfiguration(XDI_Quaternion, sampling_frequency));
+    }
+    else
+    {
+      throw std::runtime_error("No IMU device while configuring. Aborting.");
+    }
+
+    if (!m_device->setOutputConfiguration(configArray))
```
Comment (Timple) [inline src/xdainterface.cpp]: Would you mind using brackets? This reads weird:
```c++
if (!m_device->setOutputConfiguration(configArray))
  return handleError("Could not configure MTi device. Aborting.");
    return true;
```

Comment (cesar-lopez-mar) [reply src/xdainterface.cpp]: done

**Comment by cesar-lopez-mar** [comment]
ping...

**Review by Timple — APPROVED** [review]

**Pull request merged**